scripts/run_integration_tests.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
+# coding: utf-8
```

Comment:
What does this line do?

Reply:
So... it sets the default interpretation of the source code (u-strings), and to be honest, it used to be pretty useful in python2, but now I'm realizing it's deprecated and a no op as of python3.  I threw it in out of habit, but now that you've pointed it out, it's probably a good idea to remove it.  Thanks for asking!

If you're curious about the old reasoning:
    https://www.python.org/dev/peps/pep-0263/